salt/modules/monit.py

```diff
@@ -111,22 +111,29 @@ def summary(svc_name=""):
         salt '*' monit.summary <service name>
     """
     ret = {}
-    cmd = "monit summary"
+    cmd = "monit status"
```

Comment:
When I check the man pages for monit online it still shows summary as an available command. Can you elaborate why you are changing this from `summary` to `status`?